ocaml/xapi-guard/lib/varstored_interface.ml

```diff
@@ -100,6 +75,30 @@ let serve_forever_lwt rpc_fn path =
   Lwt_switch.add_hook (Some shutdown) cleanup ;
   Lwt.return cleanup
 
+let serve_forever_lwt_callback rpc_fn path _ req body =
```

Comment:
The path is only passed for debugging purposes, is it useful enough?

Reply:
The debug output says which VM we are handling the request for, which could be quite useful for debugging, I'd keep it for now.
Maybe once we add distributed tracing here then we won't need this anymore.